mak/COPY

```diff
@@ -42,10 +42,13 @@ COPY=\
 	\
 	$(IMPDIR)\core\sys\freebsd\dlfcn.d \
 	$(IMPDIR)\core\sys\freebsd\execinfo.d \
+	$(IMPDIR)\core\sys\freebsd\pthread_np.d \
+	$(IMPDIR)\core\sys\freebsd\sys\_bitset.d \
```

Comment:
Why do these modules have an underscore prepended? Just to match what the C files do?